test/groovy/TransportRequestReleaseTest.groovy

```diff
@@ -83,22 +83,56 @@ public class TransportRequestReleaseTest extends BasePiperTest {
     @Test
     public void releaseTransportRequestFailureTest() {
 
-        helper.registerAllowedMethod('sh', [Map], { Map m -> return 1 })
-
         thrown.expect(AbortException)
-        thrown.expectMessage("Cannot release Transport Request '001'. Return code from cmclient: 1.")
+        thrown.expectMessage("Something went wrong")
+
+        ChangeManagement cm = new ChangeManagement(nullScript) {
+            void releaseTransportRequest(String changeId,
+                                         String transportRequestId,
+                                         String endpoint,
+                                         String username,
+                                         String password,
+                                         String clientOpts) {
+
+                throw new ChangeManagementException('Something went wrong')
```

Comment:
a CMException is thrown but an AbortException is expected.

Reply:
Yes, the ```ChangeManagementException``` is marshalled into an AbortException in the step. AbortExceptions are handled in a special way by the framework. In case an AbortException bubbles up there is no stack trace in the log. Now we can ask: what is bad with a stack trace in the log. Basically there is nothing wrong with this. But as there is the habit I marshall the exception. We can discuss if it would be better to show the stack-trace in the log, but if we decide that I would like to do this in an other PR focusing on that.